.travis.yml

```diff
@@ -15,6 +15,10 @@ env:
     - DENO_BUILD_ARGS="is_debug=false use_allocator=\"none\" use_custom_libcxx=false use_sysroot=false"
     - DENO_BUILD_PATH=$HOME/out/Default
     - DENO_BUILD_MODE=debug
+    # ccache needs the custom LLVM to be in PATH and other variables.
+    - PATH=`pwd`/third_party/llvm-build/Release+Asserts/bin:$CARGO_PATH/bin:$PATH
```

Comment:
`` `pwd` `` here seems to really confuse travis. Maybe $HOME would work instead.

Reply:
That’s pretty whack, but I suppose it makes sense since pwd probably isn’t meant to be used when using the Travis env settings... I’ll try setting the full path